README.md

```diff
@@ -121,6 +121,8 @@ and use it on your hosts in powershell by running:
 The agent should be installed to `\Program Files\SignalFx\SignalFxAgent`, and the default configuration file
```

Comment:
Install the agent to `\Program Files\SignalFx\SignalFxAgent`

Reply:
> Install the agent to `\Program Files\SignalFx\SignalFxAgent` [delete the rest of the line]

That line is suppose to mean that the agent files will be installed to `\Program Files\SignalFx\SignalFxAgent` after running the installer script command.

Reply:
Change it to read "The installer script puts the agent files in `\Program Files\SignalFx\SignalFxAgent`. Avoid using "should". If you want users to do something, use the imperative tense (for example, "Install the agent to this directory") instead of saying "You should install the agent"). "Should" implies an option. If you want to present options, state all of them and tell users which one to use for a particular situation.